src/lib/bluetooth/bluetoothbackend.cpp

```diff
@@ -36,6 +36,14 @@ BluetoothBackend::BluetoothBackend(QObject *parent)
 {
```

Comment:
It's good that BT connection works but please see my comments on shortlog rules on other patches.